Include/pyerrors.h

```diff
@@ -47,11 +47,7 @@ PyAPI_FUNC(void) PyException_SetCause(PyObject *, PyObject *);
 PyAPI_FUNC(PyObject *) PyException_GetContext(PyObject *);
 PyAPI_FUNC(void) PyException_SetContext(PyObject *, PyObject *);
 
-/* */
-
-#define PyExceptionClass_Check(x)                                       \
-    (PyType_Check((x)) &&                                               \
-     PyType_FastSubclass((PyTypeObject*)(x), Py_TPFLAGS_BASE_EXC_SUBCLASS))
+#define PyExceptionClass_Check(exc) _PyExceptionClass_Check(_PyObject_CAST(exc))
```

Comment:
In the limited C API, you cannot use a function excluded from the limited C API. You must define _PyExceptionClass_Check() in the same header file, before the macro uses it.

Reply:
Thanks, noted.

Reply:
Moved "back" to the limited API in a6b67e65c308379f90ac28d5d541ddb1fc0e8f79